compiler/damlc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -6,161 +6,89 @@
 
 daml 1.2
 -- | MOVE Prelude DAML-LF primitives, just templates/contracts
-module DA.Internal.Template(
-    Template(ensure, signatory, observer, agreement),
-    Choice(consuming, choiceController, choice), preconsuming, nonconsuming, postconsuming, NoEvent(..),
-    stakeholder,
-    create, exercise, fetch,
-    archive, Archive(..),
-    lookupByKey, fetchByKey, exerciseByKey,
-    IsParties(toParties),
-    TemplateKey(key, maintainer)
-    ) where
-
-import GHC.Types (magic)
+module DA.Internal.Template where
+
 import DA.Internal.LF
 import DA.Internal.Prelude
 
--- NOTE: use internalCreate vs create so that people implementing Template can't override create/fetch/exercise
-
--- | Create a contract based on a template `c`.
-create : Template c => c -> Update (ContractId c)
-create = internalCreate
+class Template t where
```

Comment:
Ok, I think I dropped the comments for these 4 methods because I didn't think they were very useful. I'll restore them here and we can iterate on it later if we want.